### PR TITLE
add healthcheck to checklist

### PR DESCRIPTION
**Pull request by gidsg**
## What

add healthcheck to checklist. 

I think this is important as without one you risk putting or leaving an unhealthy instance in service.

## How to review

Describe the steps required to test the changes.

1. Preview the content locally ([see README](https://github.com/alphagov/paas-tech-docs#preview)) and check that it renders as expected.
1. Manually check that any removed or renamed anchor tags are not in use ([linkchecker](https://github.com/linkcheck/linkchecker) doesn't do this).

### CODE REVIEW

File: source/documentation/deploying_apps/production_checklist.md [hunk 1]
```diff
@@ -5,3 +5,4 @@ Before deploying an app for production use, check the following:
 1. If your app uses a PostgreSQL service, make sure that you have selected the high-availability plan (``M-HA-dedicated-9.5``) for the bound ``postgres`` service instance.
 2. You are running at least two instances of the app to ensure availability. Use ``cf scale APPNAME -i 2`` to add an extra running instance.
 3. You are prepared to use a [blue-green deployment process](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) [external link] for when the app needs to be updated or restarted (this avoids problems due to a known issue with the PaaS that can generate transient 404 errors).
+4. You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] for your app.
```
Comment (dcarley) [inline source/documentation/deploying_apps/production_checklist.md]: Thanks for the contribution.

The default `health-check-type` is `port`. Should this advice be more explicit in saying that you should use `http` and ensuring that `health-check-http-endpoint` is correct for your app?

**Comment by samcrang** [comment]
This looks good to me. I don't know if @jonathanglassman just wants to give it the final 👍 and we can merge it?

File: source/documentation/deploying_apps/production_checklist.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ Before deploying an app for production use, check the following:
 1. You have [a domain name](https://www.gov.uk/service-manual/technology/get-a-domain-name) for your service.
 1. You are running at least two instances of the app to ensure availability. Use ``cf scale APPNAME -i 2`` or amend the manifest file to add an extra running instance; see [Scaling](/#scaling) for more information.
 1. You are prepared to use a [blue-green deployment process](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) [external link] for when the app needs to be updated or restarted (this avoids problems due to a known issue with the PaaS that can generate transient 404 errors).
+1. You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] of `health-check-type` `http` and set `health-check-http-endpoint` to an appropriate endpoint for your app.
```
Comment (jonathanglassman) [inline source/documentation/deploying_apps/production_checklist.md]: I understand the content, however I would recommend phrasing it slightly differently:

"You have configured a [health check](https://docs.cloudfoundry.org/devguide/deploy-apps/healthchecks.html) [external link] by setting `health-check-type` to `http` and setting `health-check-http-endpoint` to an appropriate endpoint for your app."

@samcrang what do you think?

Comment (samcrang) [reply source/documentation/deploying_apps/production_checklist.md]: Sounds good. I'll tweak it.

**Pull request merged**